src/IntervalSets.jl

```diff
@@ -4,29 +4,73 @@ module IntervalSets
 
 using Base: @pure
 import Base: eltype, convert, show, in, length, isempty, isequal, issubset, ==, hash,
-             union, intersect, minimum, maximum, extrema, range, ⊇
+             union, intersect, minimum, maximum, extrema, range, ⊇, mean, median
```

Comment:
In Julia 0.7, `mean` and `median` are in the Statistics stdlib package. So instead of adding them here, you could add
```julia
import Compat.Statistics: mean, median
```
Just make sure that REQUIRE has an appropriate lower bound for Compat in that case.